### PR TITLE
Hackathon partial and wrap name param for speakers around with

### PR DESCRIPTION
**Pull request by sefabey**
I have no idea if wrapping name around a 'with' would break anything so please be mindful. I hope not tho.

### CODE REVIEW

**Comment by DaveParr** [comment]
Thanks a lot for doing this @sefabey , is it alright if we loop back around to this? It's probably fine, but I'm a little tight for time right now and can't review and test this locally for a few days or so.

**Comment by DaveParr** [comment]
Sorry, just saw that this is required for: https://github.com/satRdays/cardiff2019/pull/3

Will try and find time to review this change tomorrow. I need to work on the tickets tonight.

**Comment by sefabey** [comment]
Perfect. Ping me if you need me to do anything.

File: layouts/partials/team.html [hunk 1]
```diff
@@ -66,7 +66,9 @@ <h2 class="section-heading">{{ with .Site.Params.team.speakers }}{{ . | markdown
             <div class="team-member">
                 {{ $url := printf "img/team/%s" .img }}
                 <img src="{{ $url | absURL }}" class="img-responsive img-circle" alt="{{ .name }}">
-              <h3>{{ .name | markdownify }}</h3>
```
Comment (DaveParr) [inline layouts/partials/team.html]: could you revert this change please?

File: layouts/partials/hackathon.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        <h4 class="section-heading">
+            Exploring Transgender Day of Remembrance Data   
+        </h4>
+    </div>
+    <div>
+        <p>We will explore a unique dataset collated by Anna-Jayne Metcalfe for the <a href= "https://tdor.translivesmatter.info/" > TDOR  </a> website. 
+        The event will take place at {{.Site.Params.hackathon.location}} on {{ .Site.Params.hackathon.dates }} at {{.Site.Params.hackathon.times}}. The event will involve activities suitable for all levels and backgrounds so feel free to come along. Click <a href= "https://www.meetup.com/Cardiff-R-User-Group/events/256116362/" > here  </a> for further info.
+        </p>
+    </div>
+</section>
```
Comment (DaveParr) [inline layouts/partials/hackathon.html]: this looks good

File: layouts/partials/team.html (hunk 1 above)
Comment (sefabey) [reply layouts/partials/team.html]: Okay, just did. But mind that without this change, the CI failed to build when I comment out the default stuff for Speaker2.
